src/plots/cartesian/axes.js

```diff
@@ -944,9 +952,61 @@ axes.calcTicks = function calcTicks(ax, opts) {
             axes.prepTicks(mockAx, opts);
         }
 
+        if(mockAx.tickmode === 'full domain') {
+            var nt = mockAx.nticks;  // does mockAx have nitkcs?
+            if(nt === undefined) nt = 0;
+            if(nt === 0) {
+              // pass
+            } else if(nt === 1) {
+                tickVals = [0];
```

Comment:
```suggestion
                tickVals = [0.5];
```
If you just want one tick I'd put it in the middle. Not that it matters much, this is horrible whatever we choose since you can't tell the scale at all.